droid-command-line/src/main/java/uk/gov/nationalarchives/droid/command/i18n/I18N.java

```diff
@@ -81,6 +81,16 @@ public final class I18N {
      */
     public static final String ROW_PER_IDENTIFICATION = "profile.rowsPerFormat.help";
 
+    /**
+     * Writes the output as JSON.
+     */
+    public static final String JSON_OUTPUT = "profile.json.help";
+
+    /**
+     * Writes the output as XML.
+     */
+    public static final String XML_OUTPUT = "profile.xml.help";
```

Comment:
XML?

Reply:
The issue on GitHub mentioned XML output. I was going to try but then you said we'd tried it before so I stopped. Forgot about this bit :smile: